GeneratorInterface/GenFilters/test/test_MCDisplacementFilter_cfg.py

```diff
@@ -52,9 +52,10 @@
     )
 )
 
-process.select = cms.EDFilter("MCLongLivedParticles",
-    hepMCProductTag = cms.InputTag("VtxSmeared"),
-    LengCut = cms.untracked.double(100.0) ## in mm
+process.select = cms.EDFilter("MCDisplacementFilter",
+    ParticleIDs = cms.vint32(310), ## in mm
```

Comment:
Please fix the comment here. It can be confusing. 310 is the PDGID of the $K_s^{0}$.

Reply:
@perrotta, do you think that you could make a commit to remove the legend "#in mm" on line 56? I have tried to do it myself, but do not have the rights to push a commit, and @Raffaella07 is currently not available. Thank you very much in advance!

Reply:
> @perrotta, do you think that you could make a commit to remove the legend "#in mm" on line 56? I have tried to do it myself, but do not have the rights to push a commit, and @Raffaella07 is currently not available. Thank you very much in advance!

Done: apparently I have the superpowers to do it :-)